mono/tests/Makefile.am

```diff
@@ -912,7 +1009,106 @@ PLATFORM_DISABLED_TESTS += tailcall/3.exe
 PLATFORM_DISABLED_TESTS += tailcall/4.exe
 PLATFORM_DISABLED_TESTS += tailcall/8273.exe
 PLATFORM_DISABLED_TESTS += tailcall/fsharp-deeptail.exe
+PLATFORM_DISABLED_TESTS += tailcall/fsharp-deeptail-Seq.filter-length1.exe
+PLATFORM_DISABLED_TESTS += tailcall/fsharp-deeptail-Seq.filter-length2.exe
```

Comment:
Could this be written in fewer lines ? The other places too.

Reply:
It is this way for easy add/delete/merge. Adding a test is adding a line, removing a test is removing a line. No backslashes or word wrap to contend with.

Reply:
The number of lines here should go down as well. I think I have commented out disable all for ARM, and disable all for PowerPC. Those should decrease. Oh and disable all for interpreter, granted. The *.il existing style is one per line, but with merge conflicts at the end due to lack of trailing slash. I believe ease of edit/merge is valuable, but it does conflict with reading quickly, glossing over stuff.

Maybe we can if-in the .il lines? That'd be a net reduction for certain.
We also maybe use include to shrink the file. But I'm not certain. At least the disables should reduce.